resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/driver-py/Dockerfile

```diff
@@ -39,10 +39,10 @@ ENV PYSPARK_DRIVER_PYTHON python
 ENV PYTHONPATH ${SPARK_HOME}/python/:${SPARK_HOME}/python/lib/py4j-0.10.4-src.zip:${PYTHONPATH}
 
 CMD SPARK_CLASSPATH="${SPARK_HOME}/jars/*" && \
+    env | grep SPARK_JAVA_OPT_ | sed 's/[^=]*=\(.*\)/\1/g' > /tmp/java_opts.txt && \
```

Comment:
This is pretty strange and is fairly dependent on the shell being Bash instead of sh. It would be good to make this compatible with `/bin/sh` but we couldn't think of a great way to handle this sanely without using arrays.

Separately, the usage of a temporary file is also sub-par. I found that when piping the result of `env... sed` into the `readarray` command via the `<<<` operator, `readarray` treated the entire input string as a single element of the array, making it such that the resulting array only had one large JVM option. I also tried piping the results of the `sed` command into `readarray` but that also isn't working for me in my testing.

I would appreciate any feedback or suggestions from those who are experienced at shell scripting.